pkg/yang/find.go

```diff
@@ -46,7 +46,7 @@ func trimPrefix(n Node, name string) string {
 // FindGrouping finds the grouping named name in one of the parent node's
 // grouping fields.  If no parent has the named grouping, nil is returned.
 // Imported and included modules are also checked.
-func FindGrouping(n Node, name string) *Grouping {
+func FindGrouping(n Node, name string, checkedInclude map[string]bool) *Grouping {
```

Comment:
I think this should be called "seen" and commented above:

Seen is a list of modules previously seen by FindGrouping during traversal.